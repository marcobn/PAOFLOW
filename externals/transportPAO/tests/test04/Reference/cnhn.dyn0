   6   1   1
   4
   0.000000000000000E+00   0.000000000000000E+00   0.000000000000000E+00
   0.166666666666667E+00   0.000000000000000E+00   0.000000000000000E+00
   0.333333333333333E+00   0.000000000000000E+00   0.000000000000000E+00
  -0.500000000000000E+00   0.000000000000000E+00   0.000000000000000E+00
