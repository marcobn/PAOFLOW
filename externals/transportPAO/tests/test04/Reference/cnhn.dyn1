Dynamical matrix file
                                                                           
  2    4  8  4.6685830  5.0000000  5.0000000  0.0000000  0.0000000  0.0000000
           1  'C   '    10947.356792250725     
           2  'H   '    918.68110941480279     
    1    1      0.0012979      2.4334932      2.4286151
    2    1      0.5138075      2.6964246      2.4286383
    3    2      0.0010790      1.9900921      2.4286564
    4    2      0.5138317      3.1398245      2.4286925

     Dynamical  Matrix in cartesian axes

     q = (    0.000000000   0.000000000   0.000000000 ) 

    1    1
  1.02497585  0.00000000   -0.08149235  0.00000000   -0.00005345  0.00000000
 -0.08149235  0.00000000    1.33408332  0.00000000    0.00133126  0.00000000
 -0.00005345  0.00000000    0.00133126  0.00000000    0.26631171  0.00000000
    1    2
 -0.92983792  0.00000000    0.07951955  0.00000000   -0.00003591  0.00000000
  0.07955265  0.00000000   -0.70799186  0.00000000    0.00109141  0.00000000
 -0.00004770  0.00000000    0.00132764  0.00000000   -0.18764781  0.00000000
    1    3
 -0.09953319  0.00000000   -0.00039072  0.00000000   -0.00001522  0.00000000
 -0.00450611  0.00000000   -0.60540758  0.00000000    0.00038126  0.00000000
 -0.00000392  0.00000000   -0.00021332  0.00000000   -0.08510974  0.00000000
    1    4
  0.00775134  0.00000000    0.00117495  0.00000000    0.00000011  0.00000000
 -0.00344523  0.00000000   -0.05743783  0.00000000   -0.00004622  0.00000000
 -0.00000020  0.00000000   -0.00024703  0.00000000    0.01791966  0.00000000
    2    1
 -0.92983792  0.00000000    0.07955265  0.00000000   -0.00004770  0.00000000
  0.07951955  0.00000000   -0.70799186  0.00000000    0.00132764  0.00000000
 -0.00003591  0.00000000    0.00109141  0.00000000   -0.18764781  0.00000000
    2    2
  1.02443345  0.00000000   -0.08172821  0.00000000   -0.00004987  0.00000000
 -0.08172821  0.00000000    1.33463035  0.00000000    0.00123642  0.00000000
 -0.00004987  0.00000000    0.00123642  0.00000000    0.26626326  0.00000000
    2    3
  0.00772158  0.00000000    0.00118552  0.00000000   -0.00000937  0.00000000
 -0.00270841  0.00000000   -0.05744667  0.00000000    0.00031989  0.00000000
 -0.00000204  0.00000000   -0.00021486  0.00000000    0.01791200  0.00000000
    2    4
 -0.09950596  0.00000000   -0.00019567  0.00000000    0.00000279  0.00000000
 -0.00493357  0.00000000   -0.60541734  0.00000000   -0.00010344  0.00000000
  0.00000144  0.00000000   -0.00026694  0.00000000   -0.08511574  0.00000000
    3    1
 -0.09953319  0.00000000   -0.00450611  0.00000000   -0.00000392  0.00000000
 -0.00039072  0.00000000   -0.60540758  0.00000000   -0.00021332  0.00000000
 -0.00001522  0.00000000    0.00038126  0.00000000   -0.08510974  0.00000000
    3    2
  0.00772158  0.00000000   -0.00270841  0.00000000   -0.00000204  0.00000000
  0.00118552  0.00000000   -0.05744667  0.00000000   -0.00021486  0.00000000
 -0.00000937  0.00000000    0.00031989  0.00000000    0.01791200  0.00000000
    3    3
  0.11041934  0.00000000    0.00063053  0.00000000    0.00000009  0.00000000
  0.00063053  0.00000000    0.66211974  0.00000000   -0.00010501  0.00000000
  0.00000009  0.00000000   -0.00010501  0.00000000    0.04875464  0.00000000
    3    4
 -0.01446123  0.00000000    0.00046386  0.00000000    0.00000007  0.00000000
  0.00057775  0.00000000    0.00118114  0.00000000    0.00000885  0.00000000
 -0.00000009  0.00000000   -0.00006486  0.00000000    0.02518077  0.00000000
    4    1
  0.00775134  0.00000000   -0.00344523  0.00000000   -0.00000020  0.00000000
  0.00117495  0.00000000   -0.05743783  0.00000000   -0.00024703  0.00000000
  0.00000011  0.00000000   -0.00004622  0.00000000    0.01791966  0.00000000
    4    2
 -0.09950596  0.00000000   -0.00493357  0.00000000    0.00000144  0.00000000
 -0.00019567  0.00000000   -0.60541734  0.00000000   -0.00026694  0.00000000
  0.00000279  0.00000000   -0.00010344  0.00000000   -0.08511574  0.00000000
    4    3
 -0.01446123  0.00000000    0.00057775  0.00000000   -0.00000009  0.00000000
  0.00046386  0.00000000    0.00118114  0.00000000   -0.00006486  0.00000000
  0.00000007  0.00000000    0.00000885  0.00000000    0.02518077  0.00000000
    4    4
  0.11042521  0.00000000    0.00058500  0.00000000   -0.00000003  0.00000000
  0.00058500  0.00000000    0.66213451  0.00000000    0.00007572  0.00000000
 -0.00000003  0.00000000    0.00007572  0.00000000    0.04875811  0.00000000

     Dielectric Tensor:

         58.489998485884         -0.400260756819          0.000021413833
         -0.400252212044          1.106331064066         -0.000001916732
         -0.000051865818          0.000001966563          1.070279769592

     Effective Charges E-U: Z_{alpha}{s,beta}

     atom #    1
         -1.659890993863          1.083643728622          0.002341744014
          0.001114135280          0.114769841345         -0.000066970550
         -0.000012269484          0.000278669681         -0.111188412367
     atom #    2
         -1.664192938221          1.084081061375         -0.001696533774
          0.001062802723          0.114791608767          0.000060958551
         -0.000010822500          0.000327003054         -0.111184022936
     atom #    3
         -0.074134192786         -1.091858063217         -0.000652261717
         -0.001475710272         -0.095262202390          0.000034942947
         -0.000001525381         -0.000061826097          0.131227068001
     atom #    4
         -0.072942468867         -1.091977765002          0.000011876029
         -0.001388968378         -0.095253663726         -0.000030444835
          0.000000992166         -0.000074188398          0.131232675099

     Effective Charges U-E: Z_{s,alpha}{beta}

     atom #    1
     -0.158238304720E+01     -0.217231040033E-02     -0.183417750146E-05
      0.150343330198E+01      0.951088385104E-01     -0.171478194359E-04
     -0.552247179033E-01      0.297043875509E-02     -0.112003970227E+00
     atom #    2
     -0.158694904130E+01     -0.221044979761E-02      0.825537787474E-06
      0.150290782825E+01      0.951801774603E-01      0.245651437444E-04
     -0.504420290819E-01      0.262523730905E-02     -0.112052850743E+00
     atom #    3
      0.311326605686E-01     -0.972867456760E-02     -0.512060069768E-05
     -0.884220062398E+00     -0.102981074473E+00      0.143047253721E-05
     -0.146266977147E-01      0.772677589961E-03      0.135450365549E+00
     atom #    4
      0.617297613185E-01     -0.111567860415E-01      0.248979104318E-05
     -0.884490786028E+00     -0.102964125195E+00     -0.169208520445E-04
      0.164079788709E-02     -0.121510937543E-03      0.135456924276E+00

     Diagonalizing the dynamical matrix

     q = (    0.000000000   0.000000000   0.000000000 ) 

 **************************************************************************
     omega( 1) =      -5.875087 [THz] =    -195.971801 [cm-1]
 (  0.081170  0.000000  0.491180  0.000000 -0.027186  0.000000 ) 
 (  0.081417  0.000000  0.491039  0.000000 -0.012439  0.000000 ) 
 (  0.106408  0.000000  0.488363  0.000000 -0.052958  0.000000 ) 
 (  0.110785  0.000000  0.488239  0.000000  0.015359  0.000000 ) 
     omega( 2) =       1.187645 [THz] =      39.615585 [cm-1]
 (  0.008049  0.000000  0.004471  0.000000  0.156188  0.000000 ) 
 (  0.008053  0.000000  0.004494  0.000000 -0.157713  0.000000 ) 
 (  0.008004  0.000000  0.004564  0.000000  0.688639  0.000000 ) 
 (  0.008043  0.000000  0.004606  0.000000 -0.690049  0.000000 ) 
     omega( 3) =       2.767192 [THz] =      92.303602 [cm-1]
 (  0.503187  0.000000 -0.084299  0.000000  0.016324  0.000000 ) 
 (  0.503279  0.000000 -0.084280  0.000000  0.035619  0.000000 ) 
 (  0.479186  0.000000 -0.085787  0.000000 -0.017809  0.000000 ) 
 (  0.478268  0.000000 -0.085796  0.000000  0.066118  0.000000 ) 
     omega( 4) =       4.040384 [THz] =     134.772701 [cm-1]
 ( -0.022524  0.000000  0.024501  0.000000  0.518903  0.000000 ) 
 ( -0.022521  0.000000  0.024480  0.000000  0.516483  0.000000 ) 
 ( -0.020224  0.000000  0.025003  0.000000  0.484739  0.000000 ) 
 ( -0.020023  0.000000  0.024976  0.000000  0.474068  0.000000 ) 
     omega( 5) =      26.928268 [THz] =     898.230330 [cm-1]
 (  0.000010  0.000000 -0.000183  0.000000  0.244568  0.000000 ) 
 (  0.000010  0.000000 -0.000166  0.000000 -0.244526  0.000000 ) 
 (  0.000013  0.000000 -0.000326  0.000000 -0.663585  0.000000 ) 
 (  0.000008  0.000000 -0.000348  0.000000  0.663362  0.000000 ) 
     omega( 6) =      30.538228 [THz] =    1018.645635 [cm-1]
 ( -0.000002  0.000000  0.000036  0.000000 -0.054769  0.000000 ) 
 ( -0.000003  0.000000  0.000063  0.000000 -0.054807  0.000000 ) 
 ( -0.000011  0.000000  0.000120  0.000000  0.704925  0.000000 ) 
 (  0.000002  0.000000  0.000007  0.000000  0.705037  0.000000 ) 
     omega( 7) =      34.901489 [THz] =    1164.188349 [cm-1]
 (  0.056133  0.000000  0.002559  0.000000  0.000004  0.000000 ) 
 (  0.057625  0.000000  0.003904  0.000000 -0.000000  0.000000 ) 
 ( -0.708651  0.000000  0.004603  0.000000 -0.000007  0.000000 ) 
 ( -0.700905  0.000000  0.005708  0.000000  0.000000  0.000000 ) 
     omega( 8) =      35.199949 [THz] =    1174.143908 [cm-1]
 ( -0.125214  0.000000 -0.086073  0.000000 -0.000000  0.000000 ) 
 (  0.124248  0.000000  0.085531  0.000000  0.000008  0.000000 ) 
 ( -0.679556  0.000000 -0.085219  0.000000 -0.000024  0.000000 ) 
 (  0.691356  0.000000  0.084424  0.000000 -0.000005  0.000000 ) 
     omega( 9) =      38.345717 [THz] =    1279.075455 [cm-1]
 (  0.005076  0.000000 -0.331075  0.000000 -0.000011  0.000000 ) 
 ( -0.005249  0.000000  0.331513  0.000000  0.000025  0.000000 ) 
 (  0.525909  0.000000 -0.338375  0.000000 -0.000070  0.000000 ) 
 ( -0.523879  0.000000  0.338980  0.000000 -0.000004  0.000000 ) 
     omega(10) =      46.473472 [THz] =    1550.188152 [cm-1]
 (  0.335395  0.000000 -0.069735  0.000000 -0.000001  0.000000 ) 
 ( -0.335297  0.000000  0.069474  0.000000  0.000007  0.000000 ) 
 ( -0.613564  0.000000 -0.078863  0.000000 -0.000024  0.000000 ) 
 (  0.613633  0.000000  0.078558  0.000000 -0.000006  0.000000 ) 
     omega(11) =      91.324827 [THz] =    3046.268314 [cm-1]
 (  0.001332  0.000000 -0.060465  0.000000  0.000003  0.000000 ) 
 ( -0.001351  0.000000  0.060365  0.000000  0.000006  0.000000 ) 
 (  0.000139  0.000000  0.705135  0.000000 -0.000052  0.000000 ) 
 (  0.000082  0.000000 -0.703904  0.000000 -0.000079  0.000000 ) 
     omega(12) =      92.014590 [THz] =    3069.276343 [cm-1]
 ( -0.000055  0.000000  0.058792  0.000000  0.000054  0.000000 ) 
 ( -0.000078  0.000000  0.058901  0.000000  0.000058  0.000000 ) 
 ( -0.001908  0.000000 -0.704041  0.000000  0.000231  0.000000 ) 
 ( -0.002100  0.000000 -0.705260  0.000000 -0.000099  0.000000 ) 
 **************************************************************************
