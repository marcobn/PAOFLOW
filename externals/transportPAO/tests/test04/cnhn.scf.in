
poly-acetylene CnHn
   &CONTROL
      calculation = 'scf',
      restart_mode='from_scratch',
      prefix='cnhn',
      pseudo_dir = '../pseudo/',
      outdir = './'
      tprnfor=.true.
      tstress=.true.
   /
   &SYSTEM
      ibrav=8, 
      celldm(1)=4.668582996
      celldm(2)=5.0
      celldm(3)=5.0
      nat=4, 
      ntyp=2,
      ecutwfc=25.0,
      ecutrho=500.0,
   /
   &ELECTRONS
      mixing_beta = 0.7,
      conv_thr =  1.0d-10,
   /
ATOMIC_SPECIES
  C    12.011   C.pbe-rrkjus.UPF
  H   1.00794   H.pbe-rrkjus.UPF
ATOMIC_POSITIONS (angstrom)
C        0.003206499   6.011963731   5.999912409
C        1.269365328   6.661537801   5.999969702
H        0.002665788   4.916537872   6.000014312
H        1.269425141   7.756960596   6.000103577
K_POINTS {automatic}
   12 1 1    0  0  0

