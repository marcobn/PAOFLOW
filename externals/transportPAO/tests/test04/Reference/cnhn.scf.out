
     Program PWSCF v.5.0.1 starts on  1Aug2012 at 14:41:43 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote.php

     Serial version

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
     Waiting for input...
     Reading input from standard input
               file C.pbe-rrkjus.UPF: wavefunction(s)  2S 2P renormalized
               file H.pbe-rrkjus.UPF: wavefunction(s)  1S renormalized

     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum        4335     869    283               480629    42995    7821



     bravais-lattice index     =            8
     lattice parameter (alat)  =       4.6686  a.u.
     unit-cell volume          =    2543.8720 (a.u.)^3
     number of atoms/cell      =            4
     number of atomic types    =            2
     number of electrons       =        10.00
     number of Kohn-Sham states=            5
     kinetic-energy cutoff     =      25.0000  Ry
     charge density cutoff     =     500.0000  Ry
     convergence threshold     =      1.0E-10
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      =  SLA  PW   PBE  PBE ( 1 4 3 4 0)
     EXX-fraction              =        0.00

     celldm(1)=   4.668583  celldm(2)=   5.000000  celldm(3)=   5.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   5.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   5.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  0.200000  0.000000 )  
               b(3) = (  0.000000  0.000000  0.200000 )  


     PseudoPot. # 1 for C  read from file:
     ../pseudo/C.pbe-rrkjus.UPF
     MD5 check sum: a5cc68578e6c79c9731b675496d79ba7
     Pseudo is Ultrasoft, Zval =  4.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  627 points,  4 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
     Q(r) pseudized with 0 coefficients 


     PseudoPot. # 2 for H  read from file:
     ../pseudo/H.pbe-rrkjus.UPF
     MD5 check sum: 570d2d78b8586c06526eb89f0cb0f8d3
     Pseudo is Ultrasoft, Zval =  1.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of 1061 points,  2 beta functions with: 
                l(1) =   0
                l(2) =   0
     Q(r) pseudized with 0 coefficients 


     atomic species   valence    mass     pseudopotential
        C              4.00    12.01100     C ( 1.00)
        H              1.00     1.00794     H ( 1.00)

     No symmetry found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           C   tau(   1) = (   0.0012979   2.4334932   2.4286151  )
         2           C   tau(   2) = (   0.5138075   2.6964246   2.4286383  )
         3           H   tau(   3) = (   0.0010790   1.9900921   2.4286564  )
         4           H   tau(   4) = (   0.5138317   3.1398245   2.4286925  )

     number of k points=     7
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.1666667
        k(    2) = (   0.0833333   0.0000000   0.0000000), wk =   0.3333333
        k(    3) = (   0.1666667   0.0000000   0.0000000), wk =   0.3333333
        k(    4) = (   0.2500000   0.0000000   0.0000000), wk =   0.3333333
        k(    5) = (   0.3333333   0.0000000   0.0000000), wk =   0.3333333
        k(    6) = (   0.4166667   0.0000000   0.0000000), wk =   0.3333333
        k(    7) = (  -0.5000000   0.0000000   0.0000000), wk =   0.1666667

     Dense  grid:   480629 G-vectors     FFT dimensions: (  36, 180, 180)

     Smooth grid:    42995 G-vectors     FFT dimensions: (  15,  75,  75)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions         0.41 Mb     (   5395,    5)
        NL pseudopotentials             1.65 Mb     (   5395,   20)
        Each V/rho on FFT grid         17.80 Mb     (1166400)
        Each G-vector array             3.67 Mb     ( 480629)
        G-vector shells                 0.04 Mb     (   5590)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions         1.65 Mb     (   5395,   20)
        Each subspace H/S matrix        0.01 Mb     (  20,  20)
        Each <psi_i|beta_j> matrix      0.00 Mb     (     20,    5)
        Arrays for rho mixing         142.38 Mb     (1166400,   8)

     Initial potential from superposition of free atoms
     Check: negative starting charge=   -0.000192

     starting charge    9.80749, renormalised to   10.00000

     negative rho (up, down):  0.195E-03 0.000E+00
     Starting wfc are   10 randomized atomic wfcs

     total cpu time spent up to now is        4.0 secs

     Self-consistent Calculation

     iteration #  1     ecut=    25.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  2.0

     negative rho (up, down):  0.495E-04 0.000E+00

     total cpu time spent up to now is        8.2 secs

     total energy              =     -25.00033208 Ry
     Harris-Foulkes estimate   =     -25.48255703 Ry
     estimated scf accuracy    <       0.73445731 Ry

     iteration #  2     ecut=    25.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  7.34E-03,  avg # of iterations =  2.0

     negative rho (up, down):  0.369E-04 0.000E+00

     total cpu time spent up to now is       12.2 secs

     total energy              =     -25.09767430 Ry
     Harris-Foulkes estimate   =     -25.13135844 Ry
     estimated scf accuracy    <       0.06191891 Ry

     iteration #  3     ecut=    25.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  6.19E-04,  avg # of iterations =  2.0

     negative rho (up, down):  0.126E-04 0.000E+00

     total cpu time spent up to now is       16.0 secs

     total energy              =     -25.10870791 Ry
     Harris-Foulkes estimate   =     -25.11138166 Ry
     estimated scf accuracy    <       0.00455828 Ry

     iteration #  4     ecut=    25.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  4.56E-05,  avg # of iterations =  2.0

     negative rho (up, down):  0.795E-05 0.000E+00

     total cpu time spent up to now is       19.8 secs

     total energy              =     -25.10978452 Ry
     Harris-Foulkes estimate   =     -25.10977747 Ry
     estimated scf accuracy    <       0.00012093 Ry

     iteration #  5     ecut=    25.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.21E-06,  avg # of iterations =  2.3

     negative rho (up, down):  0.555E-05 0.000E+00

     total cpu time spent up to now is       23.7 secs

     total energy              =     -25.10981151 Ry
     Harris-Foulkes estimate   =     -25.10981205 Ry
     estimated scf accuracy    <       0.00000376 Ry

     iteration #  6     ecut=    25.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  3.76E-08,  avg # of iterations =  2.3

     negative rho (up, down):  0.572E-05 0.000E+00

     total cpu time spent up to now is       27.6 secs

     total energy              =     -25.10981242 Ry
     Harris-Foulkes estimate   =     -25.10981244 Ry
     estimated scf accuracy    <       0.00000011 Ry

     iteration #  7     ecut=    25.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.10E-09,  avg # of iterations =  2.9

     negative rho (up, down):  0.563E-05 0.000E+00

     total cpu time spent up to now is       31.5 secs

     total energy              =     -25.10981247 Ry
     Harris-Foulkes estimate   =     -25.10981252 Ry
     estimated scf accuracy    <       0.00000016 Ry

     iteration #  8     ecut=    25.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.10E-09,  avg # of iterations =  2.7

     negative rho (up, down):  0.536E-05 0.000E+00

     total cpu time spent up to now is       35.2 secs

     total energy              =     -25.10981249 Ry
     Harris-Foulkes estimate   =     -25.10981249 Ry
     estimated scf accuracy    <          1.4E-09 Ry

     iteration #  9     ecut=    25.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.40E-11,  avg # of iterations =  2.4

     negative rho (up, down):  0.534E-05 0.000E+00

     total cpu time spent up to now is       39.1 secs

     total energy              =     -25.10981249 Ry
     Harris-Foulkes estimate   =     -25.10981249 Ry
     estimated scf accuracy    <          2.3E-10 Ry

     iteration # 10     ecut=    25.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  2.29E-12,  avg # of iterations =  2.7

     negative rho (up, down):  0.534E-05 0.000E+00

     total cpu time spent up to now is       42.8 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 (  5395 PWs)   bands (ev):

   -20.2102 -13.4916  -8.4129  -8.3406  -7.8628

          k = 0.0833 0.0000 0.0000 (  5387 PWs)   bands (ev):

   -20.0940 -13.3881  -8.7550  -8.2622  -7.9254

          k = 0.1667 0.0000 0.0000 (  5371 PWs)   bands (ev):

   -19.7463 -13.0941  -9.8959  -8.0030  -7.8130

          k = 0.2500 0.0000 0.0000 (  5355 PWs)   bands (ev):

   -19.1707 -12.7363 -11.2545  -8.1822  -7.0717

          k = 0.3333 0.0000 0.0000 (  5372 PWs)   bands (ev):

   -18.3736 -13.3554 -11.5388  -8.5708  -6.0530

          k = 0.4167 0.0000 0.0000 (  5380 PWs)   bands (ev):

   -17.3682 -14.7669 -10.8375  -9.1979  -4.7852

          k =-0.5000 0.0000 0.0000 (  5344 PWs)   bands (ev):

   -16.3320 -15.9637 -10.0538  -9.9383  -3.5200

!    total energy              =     -25.10981249 Ry
     Harris-Foulkes estimate   =     -25.10981249 Ry
     estimated scf accuracy    <          1.4E-11 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =    -103.86785873 Ry
     hartree contribution      =      53.11244786 Ry
     xc contribution           =      -8.36942949 Ry
     ewald contribution        =      34.01502786 Ry

     convergence has been achieved in  10 iterations

     Forces acting on atoms (Ry/au):

     atom    1 type  1   force =     0.00076144    0.00157588    0.00003047
     atom    2 type  1   force =    -0.00084768   -0.00157859   -0.00000078
     atom    3 type  2   force =     0.00010488   -0.00098807   -0.00001844
     atom    4 type  2   force =    -0.00001864    0.00099078   -0.00001126

     Total force =     0.002871     Total SCF correction =     0.000007


     entering subroutine stress ...

          total   stress  (Ry/bohr**3)                   (kbar)     P=   -1.67
  -0.00000989  -0.00000040  -0.00000000         -1.46     -0.06     -0.00
  -0.00000040  -0.00001128   0.00000000         -0.06     -1.66      0.00
  -0.00000000   0.00000000  -0.00001288         -0.00      0.00     -1.89


     Writing output data file cnhn.save

     init_run     :      3.63s CPU      3.91s WALL (       1 calls)
     electrons    :     36.65s CPU     38.75s WALL (       1 calls)
     forces       :      1.97s CPU      2.02s WALL (       1 calls)
     stress       :      6.56s CPU      6.70s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.60s CPU      0.64s WALL (       1 calls)
     potinit      :      1.50s CPU      1.60s WALL (       1 calls)

     Called by electrons:
     c_bands      :      8.23s CPU      8.39s WALL (      10 calls)
     sum_band     :     10.10s CPU     10.37s WALL (      10 calls)
     v_of_rho     :     10.46s CPU     11.04s WALL (      11 calls)
     newd         :      7.37s CPU      8.37s WALL (      11 calls)
     mix_rho      :      1.14s CPU      1.17s WALL (      10 calls)

     Called by c_bands:
     init_us_2    :      0.31s CPU      0.36s WALL (     161 calls)
     cegterg      :      7.88s CPU      8.00s WALL (      70 calls)

     Called by *egterg:
     h_psi        :      7.60s CPU      7.62s WALL (     240 calls)
     s_psi        :      0.23s CPU      0.23s WALL (     240 calls)
     g_psi        :      0.05s CPU      0.05s WALL (     163 calls)
     cdiaghg      :      0.03s CPU      0.05s WALL (     233 calls)

     Called by h_psi:
     add_vuspsi   :      0.23s CPU      0.23s WALL (     240 calls)

     General routines
     calbec       :      0.36s CPU      0.36s WALL (     324 calls)
     fft          :      9.16s CPU      9.18s WALL (     180 calls)
     ffts         :      0.06s CPU      0.06s WALL (      21 calls)
     fftw         :      5.09s CPU      5.10s WALL (    2486 calls)
     interpolate  :      1.29s CPU      1.40s WALL (      21 calls)
     davcio       :      0.00s CPU      0.02s WALL (     231 calls)


     PWSCF        :    48.99s CPU        51.72s WALL


   This run was terminated on:  14:42:34   1Aug2012            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
